core/src/main/java/io/aiven/klaw/service/HAMessagingServiceI.java

```diff
@@ -0,0 +1,8 @@
+package io.aiven.klaw.service;
+
+public interface HAMessagingServiceI<T> {
```

Comment:
Type parameter is never used? for later ?

Reply:
Shoot that should not be there I had experimented with another option and it can now be removed thanks